tests/models/test_modeling_common.py

```diff
@@ -1042,7 +1110,7 @@ def test_disk_offload_without_safetensors(self):
         torch.manual_seed(0)
         base_output = model(**inputs_dict)
 
-        model_size = compute_module_sizes(model)[""]
+        model_size = compute_module_persistent_sizes(model)[""]
```

Comment:
we estimate the checkpoint size here, so should not include non-persistent buffers; technically we could pass `remove_non_persistent` to the `named_module_tensors` here but it will require a PR to accelerate so adding a function here for now
https://github.com/huggingface/accelerate/blob/200c9eb7833cfa505907f6f224ebf5a275aa6d92/src/accelerate/utils/modeling.py#L724

Reply:
Is there anything I need to do here? I'm not familiar with what you said above. 🤔